backend/model/person/schema.js

```diff
@@ -0,0 +1,11 @@
+const mongoose = require('mongoose');
+const Schema = mongoose.Schema;
+
+
+const personSchema = new Schema({
```

Comment:
Asking because I don't know. Does mongoose self assign an id? How does that id get generated and do we want to override this to be more bulletproof?

Reply:
[looks like it](http://mongoosejs.com/docs/guide.html#id)

Reply:
Ah cool. Just checking.